thorlcr/activities/soapcall/thsoapcallslave.cpp

```diff
@@ -296,15 +308,17 @@ class SoapDatasetActionSlaveActivity : public ProcessSlaveActivity, implements I
     // IThorSlaveProcess overloaded methods
     virtual void process() override
     {
+        CStatsScopedDeltaUpdater scoped(statsUpdater);
         start();
         processed = 0;
 
         processed = THORDATALINK_STARTED;
 
-        wscHelper.setown(createSoapCallHelper(this, NULL, authToken.str(), SCdataset, NULL, container.queryJob().queryContextLogger(),NULL));
+        wscHelper.setown(createSoapCallHelper(this, NULL, authToken.str(), SCdataset, NULL, contextLogger, NULL));
         wscHelper->start();
         wscHelper->waitUntilDone();
         IException *e = wscHelper->getError();
+        wscHelper.clear();
```

Comment:
I suspect this is unnecessary - the problem I was trying to fix was elsewhere, but probably harmless.